tools/recetox_aplcms/recetox_aplcms_merge_known_table.xml

```diff
@@ -0,0 +1,117 @@
+<tool id="recetox_aplcms_merge_known_table" name="RECETOX apLCMS - merge known table" version="@TOOL_VERSION@+galaxy0">
+    <description>join knowledge from aligned features and known table.</description>
+    <macros>
+        <import>macros.xml</import>
+    </macros>
+    <expand macro="creator"/>
+    <expand macro="requirements"/>
+
+    <command detect_errors="aggressive"><![CDATA[
+        Rscript -e 'source("${__tool_directory__}/utils.R")' -e 'source("${run_script}")'
+    ]]></command>
+    <configfiles>
+         <configfile name="run_script"><![CDATA[
+              metadata <- load_data_from_parquet_file('$metadata_file')
+              rt_table <- load_data_from_parquet_file('$rt_file')
+              intensity_table <- load_data_from_parquet_file('$intensity_file')
+              tolerances <- load_data_from_parquet_file('$tolerances')
+              known_table <- read_known_table('$known_table')
+
+              feature_tables <- join_tables_to_list(metadata, rt_table, intensity_table)
+
+              merged <- merge_features_and_known_table(
+                  features = feature_tables,
+                  known_table = known_table,
+                  #if $match_tol_ppm:
+                  match_tol_ppm = $match_tol_ppm,
+                  #else:
+                  match_tol_ppm = NA,
+                  #end if
+                  mz_tol_relative = get_mz_tol(tolerances),
+                  rt_tol_relative = get_rt_tol(tolerances),
+                  #if $direction.selection == "TRUE":
+                  new_feature_min_count = $direction.new_feature_min_count,
+                  #end if
+                  from_features_to_known_table = $direction.selection
+              )
+
+              #if $direction.selection == "TRUE":
+              save_known_table(merged, '$output_known_table')
+              save_pairing(merged, '$output_pairing')
+              #else:
+              save_aligned_features(merged, '$output_metadata_file', '$output_rt_file', '$output_intensity_file')
+              #end if
+         ]]></configfile>
+    </configfiles>
+
+    <inputs>
+        <param name="metadata_file" type="data" format="parquet" label="Metadata table"
+               help="TBA" />
+        <param name="rt_file" type="data" format="parquet" label="RT table"
+               help="TBA" />
+        <param name="intensity_file" type="data" format="parquet" label="Intensity table"
+               help="TBA" />
```

Comment:
@hechth any thoughts on how to briefly describe these?

Reply:
Handled in https://github.com/RECETOX/galaxytools/pull/317/commits/01d9298d8675534ac076aa5ded1fc3abf30c8064

Reply:
Actually fixed in [2c73963](https://github.com/RECETOX/galaxytools/pull/317/commits/2c739637658c753740ccddf72b1e8b81bb23bf7e), mentioned commit covered it for some other files.